tests/test_makepyfile.py

```diff
@@ -0,0 +1,274 @@
+from encodings.aliases import aliases
+from pathlib import Path
+from textwrap import dedent
+from typing import Tuple
+
+import pytest
+
+pytest_plugins = ["pytester"]
+
+
+def enc(enc):
+    try:
+        _ = "abc".encode(enc)
+    except Exception:
+        return False
+
+    return True
+
+
+all_encodings = set(filter(enc, aliases.values()))
+
+
+# @pytest.fixture(
+#     params=[
+#         "cp1252",
+#         "utf-8",
+#         "ascii",
+#         "latin1",
+#     ]
+# )
```

Comment:
```suggestion
```

Reply:
code snipped to be able to run `test_compare_make_basic_file` with an reduced set of encodings to speed things up for debugging/development of the tests